mscore/scoreview.cpp

```diff
@@ -5592,7 +5592,7 @@ void ScoreView::cmdAddRangeAnnotation()
       if (!cr)
             return;
       RangeAnnotation* range = new RangeAnnotation();
-      range->setRange(cr->segment(),cr->segment()->next()->next(),1,3);
+      range->setRange(cr->segment(), _score->tick2segment(cr->segment()->tick() + 480), selection.staffStart(), selection.staffEnd());
```

Comment:
That doe make sense, maybe something like circling a note would serve the
purpose better. We could call this segmentHighlight for now.

On Thu, Jul 7, 2016 at 3:03 AM, Marc Sabatella <notifications@github.com>
wrote:

> In mscore/scoreview.cpp
> <https://github.com/musescore/MuseScore/pull/2714#discussion_r69815795>:
>
> > @@ -5592,7 +5592,7 @@ void ScoreView::cmdAddRangeAnnotation()
> >        if (!cr)
> >              return;
> >        RangeAnnotation* range = new RangeAnnotation();
> > -      range->setRange(cr->segment(),cr->segment()->next()->next(),1,3);
> > +      range->setRange(cr->segment(), _score->tick2segment(cr->segment()->tick() + 480), selection.staffStart(), selection.staffEnd());
>
> This isn't really a single note; it's a single "beat". Could be several
> notes (eg, sixteenth notes), and it also goes the whole height of the
> staff. Eventually we need a real note highlight mode, that is specific to
> the note and not the segment. But let's get range annotation working first.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2714/files/2d0d1629d9074473b15b720529a3d2325e9c9f8e..558f1392a501f6c60d809823408b606cd6eab252#r69815795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGf7ypNQh11nys-YjMoyi7A9ryHQ2VDZks5qTB9BgaJpZM4JDs2c>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.